code/game/machinery/flasher.dm

```diff
@@ -8,40 +8,45 @@
 	base_icon_state = "mflash"
 	max_integrity = 250
 	integrity_failure = 0.4
-	light_color = COLOR_WHITE
-	light_power = FLASH_LIGHT_POWER
 	damage_deflection = 10
+	///The contained flash. Mostly just handles the bulb burning out & needing placement.
 	var/obj/item/assembly/flash/handheld/bulb
 	var/id = null
-	var/range = 2 //this is roughly the size of brig cell
-	var/last_flash = 0 //Don't want it getting spammed like regular flashes
-	var/strength = 100 //How knocked down targets are when flashed.
+	/// How far this flash reaches. Affects both proximity distance and the actual stun effect.
+	var/flash_range = 2 //this is roughly the size of a brig cell.
 
-MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/flasher, 26)
+	/// How strong Paralyze()'d targets are when flashed.
+	var/strength = 100
```

Comment:
if this is passed in directly as an argument for time make it use seconds defines

Reply:
```suggestion
	var/strength = 10 SECONDS
```